test/realm_set_test.dart

```diff
@@ -719,4 +719,52 @@ Future<void> main([List<String>? args]) async {
         ]));
     realm.write(() => testSets.objectsSet.clear());
   });
+
+  test('Set.freeze freezes the set', () {
```

Comment:
Probably it will be good to have a test for trying to write to a frozen set. You can ignore this suggestion if you don't think it's necessary to have such test.

Reply:
You would have to open a write transaction on the frozen realm. We have tests for that.